### ISSUE TITLE
Update privacy policy where necessary for online event

### ISSUE THREAD

**Issue by DorienHuijser** [OpenMRBenelux/openmrbenelux.github.io#54]
E.g., recording Zoom talks and putting materials online

**Comment by DorienHuijser** [comment]
See `faq` branch

**Comment by sofievdbos** [comment]
Any other updates necessary @DorienHuijser @jsheunis?

**Comment by DorienHuijser** [comment]
Not for me, but maybe @jsheunis indeed has to say something about it? :)

**Comment by jsheunis** [comment]
I've read through it now and I think there are a few places where the wording can improve yes.

I think we can make some changes in the interim. But for long term we should probably consider a bigger overhaul of the policy based on more recent examples. I saw a few examples come across my social media timelines and there are organisations that went the extra mile in terms of making privacy policies and codes of conduct that are very up to date with regards to aspects like GDPR, inclusivity, and catering for the peculiarities of online conferences.

Therefore, I think in the long run we should update the whole of the Privacy Policy and Code of Conduct, but for now we can make minor changes to bring the current versions more up to date with online meeting spaces.

For the minor changes to the privacy policy (which this issue is about, sorry for the detour) should I start with a PR, or has anyone started working on it?

**Comment by sofievdbos** [comment]
Nobody's started working on it, so go ahead! 😁 
Thaaaaaanks!

**Comment by DorienHuijser** [comment]
Yeah I just added a few general sentences about it, but I agree that we should take a more closer look (didn't have the patience because we were also working on the faq). So yes, please go ahead!! 😄

**Comment by sofievdbos** [comment]
Added the topic to our agenda (19 March 2021) -- to see whether (& what) we still need to add/update before the event! 🧐
